pkg/resource/edit/operations.go

```diff
@@ -126,15 +126,13 @@ search:
 }
 
 // UnprotectResource unprotects a resource, allowing it to be deleted.
-func UnprotectResource(_ *deploy.Snapshot, res *resource.State) error {
+func UnprotectResource(res *resource.State) {
 	res.Protect = false
-	return nil
 }
```

Comment:
Why is this even a named function? Probably easier to just inline the code in the places it's used then it can be seen directly that all we're doing is setting the field.

Reply:
True, changed that now.